splunk-otel-android/src/main/java/com/splunk/rum/LogToSpanBridge.java

```diff
@@ -34,6 +34,7 @@
 import io.opentelemetry.sdk.logs.data.LogRecordData;
 import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
 import java.util.concurrent.TimeUnit;
+import org.jetbrains.annotations.Nullable;
```

Comment:
VolleyHttpClientAttributesGetter uses `androidx.annotation.Nullable`, elsewhere we use `javax.annotation.Nullable`